packages/common/src/index.ts

```diff
@@ -11,8 +11,9 @@ export { getKey, splitKey } from "./util/splitKey";
 export { hrtimeBigintToSeconds } from "./util/timeUtils";
 export * from "./util/walkSync";
 export * from "./util/walkAsync";
-export { Listener, Notifier } from "./util/Notifier";
-export { TokenHatSplittingMode } from "./ide/types/Configuration";
+export { Notifier } from "./util/Notifier";
+export type { Listener } from "./util/Notifier";
```

Comment:
Is it possible to change this to

```suggestion
export { type Listener, Notifier } from "./util/Notifier";
```

Reply:
I used the VSCode quick fix :shrug:

If you want to enforce your suggested style I'd recommend enabling `@typescript-eslint/consistent-type-exports`.